### ISSUE TITLE
"Epic fail!" : Radarr won't start anymore in windows

### ISSUE THREAD

**Issue by msyquest** [Radarr/Radarr#1893]
Hello,

When I try to launch Radarr in windows server 2012 I get an error stating that the operation is not supported on this platform. It used to work and I don't know if/what I did to break it.

I tried with no success the latest version here : https://github.com/Radarr/Radarr/releases/download/v0.2.0.778/Radarr.develop.0.2.0.778.installer.exe

I took two screenshots that I hope can help identify the problem.

![radarr2](https://user-images.githubusercontent.com/6541968/29150374-2c58bfc6-7d48-11e7-997f-0ece1d7e6fc6.JPG)
![radarr](https://user-images.githubusercontent.com/6541968/29150373-2c581062-7d48-11e7-8bd0-1c0b4854a134.JPG)

Do you have an idea of what is wrong ?

**Comment by gold007eye** [comment]
I also started getting this error today (Windows 7) after the latest nightly update that happened from what I can tell around 3:28PM CST.  I have tried replacing the db file with a number of different backups, but all still give the same db corrupted error and when I tried to just delete the db file and restart Sonarr; it created a blank DB file, but will not let me log into the server.  I get the login screen but it won't actually log in.

**Comment by galli-leo** [comment]
@gold007eye Your issue is fixed on the latest nightly.

**Comment by msyquest** [comment]
@galli-leo Do you have any pointer form my issue? I tried to fin the last nightly, but I havent been able to. Do you have a link I could try?

**Comment by jamesmacwhite** [comment]
@msyquest  Restore your DB from backup and then update to latest nightly. See if that helps.

https://ci.appveyor.com/project/galli-leo/radarr-usby1/build/0.2.0.826/artifacts

Again public service announcement, don't use the nightly builds on anything you care about. This branch can be unstable and problems like this can happen.

**Comment by msyquest** [comment]
@jamesmacwhite Thank you for your help, but It didn't work.
Here is what I did after reading your post : 
 - I uninstalled Radarr, 
 - Installed this from your linl : https://ci.appveyor.com/api/buildjobs/vd4vq94ix964plsb/artifacts/Radarr.develop.0.2.0.826.installer.exe
 - Uncheck the windows service option during the installation,
 - Started Radarr,
 - See the same box poping up.

Dont know if it is revelent, I decided to try Sonarr today (new install never used it before) and it gave me the same message.

**Comment by jamesmacwhite** [comment]
@msquest I haven't seen any reports from Sonarr on this, so im thinking it might be isolated to your system. Server 2012 is supported. I'll have a quick research and see if I can provide further info for you.

**Comment by jamesmacwhite** [comment]
@msyquest So looking at info from Sonarr (upstream project of Radarr), others have reported this issue in the past, but it does look localised to something with the system Sonarr or Radarr is running on. Potentially some form of corruption somewhere.

If this has only just started happening I would have a look at what changed on the system e.g. Windows Updates, software updates etc. Check to make sure .NET framework is up to date

**Comment by msyquest** [comment]
@jamesmacwhite The issue have been there for a while, so it will be hard to pinpoint what changed on the system...

The result of _netsh.exe http show urlacl_ is : 

```
URL Reservations:
-----------------

The handle is invalid.
```

Seem like the result is the same as the reference you provided.

**Comment by jamesmacwhite** [comment]
@msyquest Looks like something borked the HTTP driver potentially. Troubleshooting that will be fun.

Potentially this might help you:

https://blogs.msdn.microsoft.com/smondal/2013/04/12/install-cannot-continue-because-some-required-components-failed/

Alternatively, you can attempt to repair and corruption with the following (as admin):

```
sfc /scannow
```

(Run DISM only if sfc displays any failed message)

```
DISM /Online /Cleanup-Image /RestoreHealth
```

At this point, this appears to be issue with your Windows install and not Radarr.

**Comment by msyquest** [comment]
@jamesmacwhite Well James, you were right, it were fun to troubleshoot, I would not have found it without your help! Thank you very much!

For reference, doing the same that were described in your las [link](https://blogs.msdn.microsoft.com/smondal/2013/04/12/install-cannot-continue-because-some-required-components-failed/) I got the error code 1058 insted of 1060 as described. Longstory short it were also a problem of HTTP (or WinHTTP) service that were not started/available. I had a website that ran on apache that were making the trouble.

So to resolve I : 
 - Changed the apache's port instance to something else than 80 (that apache had nothing to do with Radarr
 - Run this command `sc config http start= auto` that I found on this site [](https://superuser.com/questions/416712/how-to-re-install-http-windows-service)

Again Thank you for your help!
